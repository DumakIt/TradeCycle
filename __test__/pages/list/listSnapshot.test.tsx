import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { RecoilRoot } from "recoil";
import ApolloSetting from "../../../src/commons/apollo";
import ListPage from "../../../pages/list";

it("list 페이지 snapshot 테스트", () => {
  const result = render(
    <RecoilRoot>
      <ApolloSetting>
        <ListPage />
      </ApolloSetting>
    </RecoilRoot>
  );
  expect(result.container).toMatchSnapshot();
});
