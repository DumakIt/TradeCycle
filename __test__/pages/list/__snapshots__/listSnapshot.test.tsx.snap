// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list 페이지 snapshot 테스트 1`] = `
<div>
  <div
    class="css-1g8wo0y"
  >
    <button
      class="css-1jvn6fp"
    >
      상품등록
    </button>
    <div>
      <section
        class="css-1eehnxl"
      />
    </div>
  </div>
</div>
`;
